drivers/infiniband/hw/ntrdma/ntrdma_eth.c

```diff
@@ -556,13 +556,12 @@ static void ntrdma_eth_rx_fill(struct ntrdma_eth *eth)
 
 		off = start * sizeof(*rx_wqe_buf);
 		len = (pos - start) * sizeof(*rx_wqe_buf);
-		rc = ntc_request_memcpy_fenced(eth->dma_chan,
-					&eth->peer_tx_wqe_buf, off,
+		rc = ntc_memcpy(&eth->peer_tx_wqe_buf, off,
```

Comment:
maybe add direct function in case other still want to use dma for this, make sure that all the transactions on the ring should use the same method (cpu #/ dma channel  etc.)